src/bindings/python/src/openvino/runtime/opset9/ops.py

```diff
@@ -0,0 +1,762 @@
+    :param filters: The node providing filters tensor.
+    :param strides: The distance (in pixels) to slide the filter on the feature map over the axes.
+    :param pads_begin: The number of pixels to add to the beginning along each axis.
+    :param pads_end: The number of pixels to add to the end along each axis.
+    :param dilations: The distance in width and height between elements (weights) in the filter.
+    :param mask: The node providing modulation scalar (mask) tensor.
+    :param auto_pad: The type of padding. Range of values: explicit, same_upper, same_lower, valid.
+    :param group: The number of groups which both output and input should be split into.
+    :param deformable_group: The number of groups which deformable values and output should be split
+                             into along the channel axis.
+    :param bilinear_interpolation_pad: The flag that determines the mode of bilinear interpolation
+                                               execution.
+    :param name: The optional new name for output node.
+    :return: New node performing deformable convolution operation.
+    """
+    if mask is None:
+        inputs = as_nodes(data, offsets, filters)
+    else:
+        inputs = as_nodes(data, offsets, filters, mask)
+
+    return _get_node_factory_opset9().create(
+        "DeformableConvolution",
+        inputs,
+        {
+            "strides": strides,
+            "pads_begin": pads_begin,
+            "pads_end": pads_end,
+            "dilations": dilations,
+            "auto_pad": auto_pad,
+            "group": group,
+            "deformable_group": deformable_group,
+            "bilinear_interpolation_pad": bilinear_interpolation_pad
+        },
+    )
+
+
+@nameable_op
+def adaptive_avg_pool(
+        data: NodeInput,
+        output_shape: NodeInput
+) -> Node:
+    """Return a node which performs AdaptiveAvgPool operation.
+
+    :param data: The list of input nodes
+    :param output_shape: the shape of spatial dimentions after operation
+    :return: The new node performing AdaptiveAvgPool operation on the data
+    """
+    inputs = as_nodes(data, output_shape)
+    return _get_node_factory_opset9().create("AdaptiveAvgPool", inputs)
+
+
+@nameable_op
+def adaptive_max_pool(
+        data: NodeInput,
+        output_shape: NodeInput,
+        index_element_type: str = "i64"
+) -> Node:
+    """Return a node which performs AdaptiveMaxPool operation.
+
+    :param data: The list of input nodes
+    :param output_shape: the shape of spatial dimentions after operation
+    :param index_element_type: Type of indices output.
+    :return: The new node performing AdaptiveMaxPool operation on the data
+    """
+    inputs = as_nodes(data, output_shape)
+
+    attributes = {
+        "index_element_type": index_element_type,
+    }
+
+    return _get_node_factory_opset9().create("AdaptiveMaxPool", inputs, attributes)
+
+
+@nameable_op
+def multiclass_nms(
```

Comment:
Again: Please keep only new `multiclass_nms` here. There is no need to copy all other ops as there are no actual changes to them.